apache-solr/dashboards/apache-solr-cluster-overview.libsonnet

```diff
@@ -0,0 +1,1680 @@
+          {
+            id: 'custom.hidden',
+            value: true,
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'zk_host',
+        },
+        properties: [
+          {
+            id: 'custom.hidden',
+            value: true,
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'Value',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'Status',
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'instance',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'Instance',
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'shard',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'Shard',
+          },
+        ],
+      },
+      {
+        matcher: {
+          id: 'byName',
+          options: 'collection',
+        },
+        properties: [
+          {
+            id: 'displayName',
+            value: 'Collection',
+          },
+        ],
+      },
+    ],
+  },
+  options: {
+    cellHeight: 'sm',
+    footer: {
+      countRows: false,
+      fields: [],
+      reducer: [
+        'sum',
+      ],
+      show: false,
+    },
+    showHeader: true,
+  },
+  pluginVersion: '9.4.3',
+};
+
+local replicaStatePanel = {
+  datasource: {
+    type: 'prometheus',
+    uid: '${prometheus_datasource}',
+  },
+  description: 'Shows the total percent of running shards in the cluster.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      max: 100,
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'red',
+            value: null,
+          },
+          {
+            color: 'yellow',
+            value: 80,
+          },
+          {
+            color: 'green',
+            value: 95,
+          },
+        ],
+      },
+      unit: 'percent',
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'value',
+    wideLayout: true,
+  },
+  pluginVersion: '9.4.3',
+  targets: [
+    {
+      disableTextWrap: false,
+      expr: '100 * sum(solr_collections_replica_state{job=~"$job", solr_cluster=~"$solr_cluster"})  / count(solr_collections_replica_state{job=~"$job", solr_cluster=~"$solr_cluster"})',
+      fullMetaSearch: false,
+      includeNullMetadata: true,
+      instant: false,
+      legendFormat: '__auto',
+      range: true,
+      useBackend: false,
+    },
+  ],
+  title: 'Running shards',
```

Comment:
It looks like this panel is repeated twice? Not sure if it's different. If it is we should probably change the name.
<img width="1679" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/38274348/2e521dc5-6ca5-4351-ad3d-016242ad6ea5">

Reply:
First is shards, second should be replicas. Fixed!